brainlit/utils/session.py

```diff
@@ -31,6 +31,7 @@ class NeuroglancerSession:
         url: Precompued path either to a file URI or url URI. Defaults to mouselight brain1.
         mip: Resolution level to pull and push data at. Defaults to 0, the highest resolution.
         url_segments: Precomputed path to segmentation data. Optional, default None.
+        use_https: Whether to use a potentially read-only cached version of the data. Optional, default True.
```

Comment:
In your PR comment, you said that this addition allows for reading public aws data, but your description says something else, can you please reconcile these two descriptions?

Reply:
Descriptions resolved.